sdcm/cluster.py

```diff
@@ -1335,6 +1384,16 @@ def disable_client_encrypt(self):
         self.stop_scylla()
         self.start_scylla()
 
+    def prepare_file_for_archive(self, fileslist):
+        tmp_dir = '/tmp/%s' % self.name
+        self.remoter.run('mkdir %s' % tmp_dir)
+        for f in fileslist:
+            tmp_store_dir = os.path.join(tmp_dir, os.path.dirname(f)[1:])
```

Comment:
This is tricky, the one that will use this method won't understand which directory structure it will create.
Currently, the method is not general enough to put this in the base class.
Either generalize it, or make it local to where it is used.

Reply:
Do some changes, and add comments.